vocdoni/bridge-ui
# Feature/vot 93 landing page with must haves

## Short description
This PR changes the Landing Page's design to the Aragon look and feel design. 

## How can it be tested
Open the staging version of the website and check that the design 

- is aligned with the design on Figma
- satisfies all criteria specified by the corresponding issue

## Tracker ID
https://linear.app/aragon/issue/VOT-93/landing-page-with-must-haves
